src/Containers/OhmmsPETE/OhmmsMatrix.h

```diff
@@ -149,17 +149,17 @@ class Matrix
    *  Hacky but so is just making the matrix n x (n + padding) to handle row alignment.
    *  This is unoptimized.
    */
-  template<typename CONTAINER>
-  void assignUpperRight(const CONTAINER& other)
+  template<class T_FROM, typename ALLOC_FROM>
+  void assignUpperLeft(const Matrix<T_FROM, ALLOC_FROM>& from)
```

Comment:
Less dangerous now.  We should really have a unit test.

Reply:
Unit test added.